//tests/core/race:race_test fails on Windows

On our new Bazel CI Windows VM image, this test started to fail. The relevant error message seems to be: "ThreadSanitizer failed to allocate 0x000000531000 (5443584) bytes at 0x200df7c76c000 (error code: 87)".

I found an issue report in the Go repo that matches: https://github.com/golang/go/issues/46099. Unfortunately no solution or idea what it could be there. :/

Should we disable the test on Windows?

```
Executing tests from //tests/core/race:race_test
-----------------------------------------------------------------------------
--- FAIL: Test (16.44s)
    --- FAIL: Test/cmd_attr (2.28s)
        race_test.go:269: running: bazel run //:racy_cmd_race_mode -- -wantrace=true
        race_test.go:288: wanted data race; command failed with: exit status 66
            stderr:
            Loading: 
            Loading: 0 packages loaded
            Analyzing: target //:racy_cmd_race_mode (0 packages loaded, 0 targets configured)
            INFO: Analyzed target //:racy_cmd_race_mode (0 packages loaded, 313 targets configured).
            INFO: Found 1 target...
            [0 / 5] [Prepa] BazelWorkspaceStatusAction stable-status.txt
            [3 / 6] checking cached actions
            Target //:racy_cmd_race_mode up-to-date:
              bazel-out/x64_windows-fastbuild-ST-fcb5cf22ec9a/bin/racy_cmd_race_mode_/racy_cmd_race_mode.exe
            INFO: Elapsed time: 2.235s, Critical Path: 1.53s
            INFO: 1 process: 1 internal.
            INFO: Build completed successfully, 1 total action
            INFO: Running command line: bazel-out/x64_windows-fastbuild-ST-fcb5cf22ec9a/bin/racy_cmd_race_mode_/racy_cmd_race_mode.exe '-wantrace=true'
            INFO: Build completed successfully, 1 total action
            ==6436==ERROR: ThreadSanitizer failed to allocate 0x000000531000 (5443584) bytes at 0x200df7c76c000 (error code: 87)
    --- FAIL: Test/cmd_feature (2.98s)
        race_test.go:269: running: bazel run --@io_bazel_rules_go//go/config:race //:racy_cmd -- -wantrace=true
        race_test.go:288: wanted data race; command failed with: exit status 66
            stderr:
            Loading:  (0 packages loaded)
            Loading: 0 packages loaded
            INFO: Build option --@io_bazel_rules_go//go/config:race has changed, discarding analysis cache.
            Analyzing: target //:racy_cmd (0 packages loaded, 0 targets configured)
            INFO: Analyzed target //:racy_cmd (0 packages loaded, 7251 targets configured).
            INFO: Found 1 target...
            [0 / 4] [Prepa] BazelWorkspaceStatusAction stable-status.txt
            [8 / 10] [Prepa] GoCompilePkg racy_cmd.a
            Target //:racy_cmd up-to-date:
              bazel-bin/racy_cmd_/racy_cmd.exe
            INFO: Elapsed time: 2.924s, Critical Path: 2.22s
            INFO: 3 processes: 1 internal, 2 local.
            INFO: Build completed successfully, 3 total actions
            INFO: Running command line: bazel-bin/racy_cmd_/racy_cmd.exe '-wantrace=true'
            INFO: Build completed successfully, 3 total actions
            ==6444==ERROR: ThreadSanitizer failed to allocate 0x000000531000 (5443584) bytes at 0x200dfd206c000 (error code: 87)
FAIL
cleanup error: remove C:\b\zat3l52b\bazel_testing\bazel_go_test\main: The process cannot access the file because it is being used by another process.
```

Full log: https://storage.googleapis.com/bazel-testing-buildkite-artifacts/3459556c-551e-4212-9299-2a0e8c31b622/tests%5Ccore%5Crace%5Crace_test%5Cattempt_1.log

Job: https://buildkite.com/bazel-testing/rules-go-golang/builds/18744#3459556c-551e-4212-9299-2a0e8c31b622

--- a/.bazelci/presubmit.yml
+++ b/.bazelci/presubmit.yml
@@ -305,6 +305,7 @@ tasks:
     - "-//tests/core/go_test:data_test"
     - "-//tests/core/go_test:pwd_test"
     - "-//tests/core/nogo/coverage:coverage_test"
+    - "-//tests/core/race:race_test" # fails on Windows due to upstream bug, see issue #2911
     - "-//tests/core/stdlib:buildid_test"
     - "-//tests/examples/executable_name:executable_name"
     - "-//tests/integration/gazelle:gazelle_test" # exceeds command line length limit

